tools/deployment/osquery.example.conf

```diff
@@ -17,21 +15,6 @@
     // This is very helpful to prevent system performance impact when scheduling
     // large numbers of queries that run a smaller or similar intervals.
     //"schedule_splay_percent": "10",
-
-    // A filesystem path for disk-based backing storage used for events and
-    // query results differentials. See also 'use_in_memory_database'.
-    //"database_path": "/var/osquery/osquery.db",
-
-    // Comma-delimited list of table names to be disabled.
-    // This allows osquery to be launched without certain tables.
-    //"disable_tables": "foo_bar,time",
-
-    // Comma-delimited list of table names to be enabled.
-    // This allows osquery to be launched with certain tables only.
-    //"enable_tables": "foo_bar,time",
-
-    // Select the osquery logging plugin.
-    "logger_plugin": "filesystem"
```

Comment:
This one is a FLAG, not a CLI_FLAG

Reply:
You're right, although this is incorrectly a FLAG (because you cannot change the logger plugin at runtime).
I'll revert this part.

Reply:
You think it's incorrect? I feel like this sort of thing _should_ be able to come in via config. But if we don't support that, then we should remove it from here and fix the flag.

Reply:
So, there are several other FLAGs that should be changed to CLI_FLAGs to reflect the current intent and behavior, around the ability to actually act on the changed value or not, but I haven't yet go through them all.

As for this one, yeah the logger plugin is only initialized once at osquery start.
While I don't in general see an issue in supporting a logger system that can be reconfigured, I think doing that in osquery is not exactly easy if logs are being buffered. I mean I'm mostly wondering how it looks like switching from one logger to the other while there's activity.